### DISCUSSION TITLE
Should we release 0.70.14?

### DISCUSSION THREAD

**Discussion by fortmarek** [reactwg/react-native-releases#84] [Patches]
Current Release: [0.70.13](https://github.com/facebook/react-native/releases/tag/v0.70.13)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/f14da92d6c082ed21540ef2e1da6622d27a5400c
2. https://github.com/facebook/react-native/commit/462d93d9c18b453f0c14c15cf44c8d1de280edb9
3. https://github.com/facebook/react-native/commit/63884b48882f737fa652e07227800c9baea6197c
4. https://github.com/facebook/react-native/commit/ce39931bc2b02f13cbc5751ba4d4a6dbc07bc91a

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
release crew specific improvement to merge asap -> https://github.com/facebook/react-native/pull/38892

doesn't need a new patch, but merge for consistency across all versions in support window + main

**Comment by stratus1011** [comment]
Is it possible to get this change pulled into 0.70.14 https://github.com/facebook/react-native/pull/38256/commits/f14da92d6c082ed21540ef2e1da6622d27a5400c ? Currently blocked on running on Android 14 and the path to upgrade to 71 or 72 is posing a lot of challenges for our codebase.

**Reply by liamjones** [reply]
If it helps; you only need to target API level 33 this year (Android 13), not 34: https://support.google.com/googleplay/android-developer/answer/11926878?hl=en

Also, if you have an app affected by this you can log in to the Google Play Console, navigate to the app then 'Policy Status' at the bottom of the left nav and apply for an extension to Nov 1st if you need more time.

**Comment by erinleigh90** [comment]
Can we backport [facebook/react-native@462d93d](https://github.com/facebook/react-native/commit/462d93d9c18b453f0c14c15cf44c8d1de280edb9) to 0.70.14? We would like to support customers on supported versions of React Native, and would like to remove the unset-value@1.0.0 dependency.

**Comment by cipolleschi** [comment]
Fixes for Xcode 15

https://github.com/facebook/react-native/commit/21763e85e39e17a19a1cf7a9026ef74517464749
https://github.com/facebook/react-native/commit/0dbd621c598e3ba7a203ec41bb70ce395ad1d62c
https://github.com/facebook/react-native/commit/8a5b2d673502037731ee6bc40fc64cdd22139011

**Reply by fortmarek** [reply]
Already picked ✅

**Reply by joe-sam** [reply]
Update fix for the Xcode 15 patch fixes [facebook/react-native@63884b4](https://github.com/facebook/react-native/commit/63884b48882f737fa652e07227800c9baea6197c)

**Comment by cipolleschi** [comment]
Cherry-pick: https://github.com/facebook/react-native/commit/ce39931bc2b02f13cbc5751ba4d4a6dbc07bc91a

They released ActiveSupport 7.1.0 which breaks Cocoapods. 😮‍💨
This should fix the issue for all our users as they should use Bundler.

**Comment by ernestbies** [comment]
@fortmarek Do you have any approximate release date of 0.70.14? 

[facebook/react-native@ffbec5b](https://github.com/facebook/react-native/commit/f14da92d6c082ed21540ef2e1da6622d27a5400c) is really important because it's adding support for Android 14 ;)

**Reply by cipolleschi** [reply]
The pick can't be made easily as Android does not build with that commit on 0.70. Our [CI failed](https://app.circleci.com/pipelines/github/facebook/react-native/34198/workflows/01bc445f-ed23-4dd4-8d50-a2561459e83b/jobs/1128094/parallel-runs/0/steps/0-117?invite=true#step-117-26894_56) due to that change. There are probably other changes required to make that commit pickable.

I suggest to migrate to 0.71 if you need that.

**Reply by cortinico** [reply]
Correct, if you need to support the latest Android version, please update to the latest stable version of React Native. We won't be able to do this pick

**Comment by lunaleaps** [comment]
0.70.14 has been released. You can discuss picks for 0.70.15 here: https://github.com/reactwg/react-native-releases/discussions/92
